### ISSUE TITLE
Crash on login (CLI)

### ISSUE THREAD

**Issue by luksamuk** [keybase/client#19665]
I cannot login using `keybase login`.
I am using Void Linux currently, but since `kbfs` is out-of-date on repos, I've installed `keybase` from source. However, the login process only gives me an "ERROR EOF" as output, after I type my username and press Enter.

Looking at the service logs, it seems like there is a "slice out of bounds" error somewhere.

my log id: 7cc0a52d1bb51e3903d56b1c

**Comment by luksamuk** [comment]
Is there any news on this? I've been unable to access my files on Keybase for weeks.

**Comment by luksamuk** [comment]
Nevermind this -- I found the solution. I'll leave what I did here for future reference.

Turns out that setting the variable `XDG_RUNTIME_DIR` was affecting my experience on Void Linux. Some tutorials recommended it, but this directory defaults to `/run/user/1000` on Void.

I just removed this setup and my keyring was happy again. Plus, Keybase has been updated to 4.6.0 five days ago from when I wrote this message, so everything is good to go; I just had to create my keybase directory and add my user's permissions to mount it.
